<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTServerClass</class>
 <widget class="QMainWindow" name="QTServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>怪物猎人Online$</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CMyTableWidget" name="tableWidget">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>27</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_property">
    <property name="title">
     <string>挂机设置</string>
    </property>
    <addaction name="actionOption"/>
   </widget>
   <widget class="QMenu" name="menu_account">
    <property name="title">
     <string>账号设置</string>
    </property>
    <addaction name="actionAddAccount"/>
    <addaction name="actionImportAcc"/>
    <addaction name="actionExportAcc"/>
   </widget>
   <addaction name="menu_property"/>
   <addaction name="menu_account"/>
  </widget>
  <action name="actionOption">
   <property name="text">
    <string>屬性設置</string>
   </property>
  </action>
  <action name="actionAddAccount">
   <property name="text">
    <string>添加账号</string>
   </property>
  </action>
  <action name="actionImportAcc">
   <property name="text">
    <string>导入账号</string>
   </property>
  </action>
  <action name="actionExportAcc">
   <property name="text">
    <string>导出账号</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyTableWidget</class>
   <extends>QTableWidget</extends>
   <header>cmytablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtserver.qrc"/>
 </resources>
 <connections/>
</ui>
